lib/migrations.py

```diff
@@ -47,8 +47,21 @@ def __init__(self, config, cur_ver, new_ver):
 
         self.load_migrations()
 
+    def check_config_migration_dir(self, dir):
```

Comment:
Since the `exist_ok=True` flag is used, might as well just have `load_migrations()` just call `os.makedirs(config_dir, exist_ok=True)` instead of wrapping it in a method.